core/ir/aarch64/encode.c

```diff
@@ -210,24 +210,27 @@ const reg_id_t d_r_reg_id_to_virtual[] = {
     DR_REG_NULL, /* DR_REG_NULL */
```

Comment:
The DR_REG_V0 didn't exist in the last release, so we don't need to list this as a breaking change and bump the version to 10.94, right?  Maybe we're still technically supposed to bump to 10.94.  But we're about to go to 11.0 so maybe it's fine.

Reply:
Let's increase the version to 10.94. Done.